x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/analysis/index/IndexResolverTests.java

```diff
@@ -149,7 +149,7 @@ public void testMetaFieldsAreIgnored() throws Exception {
         addFieldCaps(fieldCaps, "text", "keyword", true, true);
 
         String wildcard = "*";
-        IndexResolution resolution = mergedMappings(wildcard, new String[] { "index" }, fieldCaps);
+        IndexResolution resolution = mergedMappings(wildcard, new String[] { "org/elasticsearch/xpack/sql/index" }, fieldCaps);
```

Comment:
This change was unintended (an editor unintended action) and will be reverted, along with all similar others.